docs/data/material/getting-started/templates/dashboard/components/PageViewsChart.tsx

```diff
@@ -68,30 +68,33 @@ export default function PageViewsChart() {
             },
             {
               id: 'downloads',
-              curve: 'linear',
```

Comment:
I ended up ditching the linear curve, this smooth one feels better. 🤔 
![image](https://github.com/mui/material-ui/assets/72460825/e4a1f53d-fd0d-4d8f-b8d7-8994d6881dee)

Reply:
I was thinking about this chart yesterday while I was working on the stat cards. My impression is that it feels like it's too bloated. Should we try to replace it for [bars](https://mui.com/x/react-charts/bar-demo/#simplebarchart)?